Gopkg.toml

```diff
@@ -40,7 +40,7 @@
 
 [[constraint]]
```

Comment:
@LucasRoesler anything significant here that would cause the e2e tests to start to fail?

Reply:
https://github.com/openfaas/faas-provider/compare/0.14.0...v0.15.1

Reply:
nothing obvious to me